__tests__/api.test.js

```diff
@@ -31,13 +32,38 @@ describe("Full API Test Suite", () => {
           });
         });
     });
-    test(`404: /api/topic returns a error with message "Route not found" when given a bad path`, () => {
+  });
+  describe("/api", () => {
+    test("200: /api fetches all api endpoints available to the user", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Object.keys(body).length).toEqual(
+            Object.keys(endpointData).length
+          );
+          Object.keys(endpointData).forEach((key) => {
+            expect(body).toHaveProperty(key);
+          });
+        });
```

Comment:
since we have required in the endpoint data there isn't much point iterating over it, we can just assert with expect the endpoint response to look like the required in data.